helm/netobserv/templates/flows.netobserv.io_flowmetrics.yaml

```
@@ -0,0 +1,320 @@
+              Cannot be updated.
+              In CamelCase.
+              More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: |-
+              FlowMetricSpec defines the desired state of FlowMetric
+              The provided API allows you to customize these metrics according to your needs.<br>
+              When adding new metrics or modifying existing labels, you must carefully monitor the memory
+              usage of Prometheus workloads as this could potentially have a high impact. Cf https://rhobs-handbook.netlify.app/products/openshiftmonitoring/telemetry.md/#what-is-the-cardinality-of-a-metric<br>
+              To check the cardinality of all NetObserv metrics, run as `promql`: `count({__name__=~"netobserv.*"}) by (__name__)`.
+            properties:
+              buckets:
+                description: A list of buckets to use when `type` is "Histogram".
+                  The list must be parsable as floats. When not set, Prometheus default
+                  buckets are used.
+                items:
+                  type: string
+                type: array
+              charts:
+                description: Charts configuration, for the OpenShift Console in the
+                  administrator view, Dashboards menu.
+                items:
+                  description: Configures charts / dashboard generation associated
+                    to a metric
+                  properties:
+                    dashboardName:
+                      default: Main
+                      description: Name of the containing dashboard. If this name
+                        does not refer to an existing dashboard, a new dashboard is
+                        created.
+                      type: string
+                    queries:
+                      description: |-
+                        List of queries to be displayed on this chart. If `type` is `SingleStat` and multiple queries are provided,
+                        this chart is automatically expanded in several panels (one per query).
+                      items:
+                        description: Configures PromQL queries
+                        properties:
+                          legend:
+                            description: |-
+                              The query legend that applies to each timeseries represented in this chart. When multiple timeseries are displayed, you should set a legend
+                              that distinguishes each of them. It can be done with the following format: `{{ Label }}`. For example, if the `promQL` groups timeseries per
+                              label such as: `sum(rate($METRIC[2m])) by (Label1, Label2)`, you may write as the legend: `Label1={{ Label1 }}, Label2={{ Label2 }}`.

```

Looks like escaping `{{` is needed here:

```suggestion
                              that distinguishes each of them. It can be done with the following format: {{`{{ Label }}`}}. For example, if the `promQL` groups timeseries per
                              label such as: `sum(rate($METRIC[2m])) by (Label1, Label2)`, you may write as the legend: {{`Label1={{ Label1 }}, Label2={{ Label2 }}`}}.

```